src/app/modules/newsfeed/poster/poster.component.ts

```diff
@@ -48,6 +49,13 @@ export class PosterComponent {
     this.minds = window.Minds;
   }
 
+  ngOnInit() {
+    this.client.get('api/v2/search/suggest/user')
```

Comment:
I decided to keep this route in order to pre-populate the list with usernames so the user doesn't have to type anything for the initial list.

In my mind, this can be prepopulated with the user's network of friends and only do a deeper search via the `v2/search/suggest` route after they type three or more characters.